support/cas-server-support-inwebo-mfa/build.gradle

```diff
@@ -0,0 +1,19 @@
+description = "Apereo CAS Inwebo MFA Support"
+dependencies {
+    api project(":api:cas-server-core-api-webflow")
+
+    implementation libraries.springwsclient
```

Comment:
Remove in favor of springws

Reply:
Not sure on this. See my previous comment.